tests/manager/manager-executeBulkWrite_error-011.phpt

```diff
@@ -12,16 +12,17 @@ MongoDB\Driver\Manager::executeBulkWrite() BulkWriteException inherits labels fr
 require_once __DIR__ . "/../utils/basic.inc";
 
 $manager = new MongoDB\Driver\Manager(URI);
+$server = $manager->selectServer(new \MongoDB\Driver\ReadPreference('primary'));
```

Comment:
This test requires a replica set and wouldn't run on a sharded cluster. Is this necessary?

Reply:
Technically, you're correct. Looking at the test, I'd think that it also applies to sharded clusters backed by replica set since they also support transactions, correct?

Reply:
I've changed this on a bunch of tests that were currently only running on replica sets. These tests are now either unrestricted or they run on replica sets and sharded clusters with replica sets.